eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -222,8 +222,13 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
             case None =>
               // in all other cases we need to be ready for any type of closing
               // TODO: should we wait for an acknowledgment from the watcher?
-              blockchain ! WatchSpent(self, data.commitments.commitInput.outPoint.txid, data.commitments.commitInput.outPoint.index.toInt, data.commitments.commitInput.txOut.publicKeyScript, BITCOIN_FUNDING_SPENT)
-              blockchain ! WatchLost(self, data.commitments.commitInput.outPoint.txid, nodeParams.minDepthBlocks, BITCOIN_FUNDING_LOST)
+              // avoid herd effect at startup because watch-spent are intensive in terms of rpc calls to bitcoind
+              val watchSpentDelay = Random.nextLong(1 + nodeParams.watchSpentWindow.toSeconds).seconds
```

Comment:
Why do you add 1?

Reply:
Can't have a zero duration unfortunately. Pretty ugly...

Reply:
But you *can* have a zero duration with your current code!
You need to do `1 + Random.nextLong(nodeParams.watchSpentWindow.toSeconds).seconds` if you want to ensure a non-zero duration

Reply:
My bad, zero duration are ok but `Random.nextLong(0)` is *verboten*.

Reply:
With 5dcdf5e, only the router watches are delayed. Since it's less sensitive than channel watches, I've chosen a non-zero default and replaced `+ 1` guard by a warning in the documentation.

Reply:
Actually, better to have a conf check.... Done in 5d3c1506a.